UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Extensions/DateTimeExtensionsTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NUnit.Framework;
+using System;
+using UKHO.ExchangeSetService.API.Extensions;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Extensions
+{
+    [TestFixture]
+    public class DateTimeExtensionsTest
+    {
+        [Test]
+        public void CheckIsValidDate()
+        {
+            bool isValidDate = DateTimeExtensions.IsValidDate("08Sep2021", out DateTime fakeFolderDateTime);
+            Assert.IsTrue(isValidDate);
+        }
+    }
+}
```

Comment:
Great to see this refactoring!

It would be good practice to add a test for an invalid date as well.

Would it be possible to add tests for the IsValidRfc1123Format method?

Reply:
Yes, added some unit test as mention. Thanks!